scripts/jenkins-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+[ -d venv ] || virtualenv venv
+pip install -e .
+
+if [ "$ENVIRONMENT" = "Select one" -o "$ENVIRONMENT" = "" ]; then
+  >&2 echo "You must select an environment."
+  exit 1
+fi
+
+if [ "$APPLICATION_NAME" = "Select one" -o "$ENVIRONMENT" = ""]; then
+  >&2 echo "You must select an application name."
+  exit 1
+fi
+
+if [ "$ENVIRONMENT" = "preview" ]; then
```

Comment:
At some point we have to have the knowledge of which stages are associated with which accounts. I'd rather that be recorded somewhere in git, the jenkins credentials file will probably not be.